feature-libs/organization/administration/core/services/organization-page-meta.resolver.ts

```diff
@@ -106,4 +108,8 @@ export class OrganizationPageMetaResolver
   resolveTitle(): Observable<string> {
     return this.contentPageMetaResolver.resolveTitle();
   }
+
+  resolveRobots(): Observable<PageRobotsMeta[]> {
+    return this.contentPageMetaResolver.resolveRobots();
```

Comment:
Agreed. My initial thought was that this class could benefit from any customisations done to the `ContentPageMetaResolver`, but if we'd like to achieve that we should use inheritance rather than composition.